2_adventures/001/drammopo/06_retrospectives/README.md

```diff
@@ -0,0 +1,8 @@
+## RubySteps 21-Day Challenge: Day 6
+
+No code today.
+
+### Activities for the day
+* Read [21-Day restrosepctives](https://github.com/RubySteps/21-day-challenge/issues?utf8=%E2%9C%93&q=label%3Aretrospective+) daily
+* Figure out how to get the upstream history...
```

Comment:
`git remote add upstream git@github.com/rubysteps/21-day-challenge.git` (I prefer the ssh link... you may use the other clone links, as you see fit.

Then do a `git checkout master` and then `git pull upstream master`.

You can then at the terminal do something like `git log --patch` and see everything coming in with changes.  Only those changes that have been merged, not simply sitting in queue as a pull request.